scripts/microbe_stage/tutorial/setup.as

```diff
@@ -0,0 +1,43 @@
+// The biome in which the tutorial is played.
+const str TUTORIAL_BIOME = "default"
+
+void setupBackground(CellStageWorld@ world){
```

Comment:
The normal microbe_stage setup takes care of biome and camera setup so it looks like this file can also be entirely deleted.